### PR TITLE
Add initial implementation of obfuscation and translation

### PR DESCRIPTION
**Pull request by danielfc**
- word obfuscation and translation initial implementation
- fix charset issues in samples

In order to run the initial implementation, execute:

``` shell
$ cd server/modules/blendsubs/ (path dependent yet)
$ node blendsubs.js
```

### CODE REVIEW

File: server/modules/blendsubs/word-obfuscation/word-obfuscation.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = function(word) {
```
Comment (danielfc) [inline server/modules/blendsubs/word-obfuscation/word-obfuscation.js]: I think we don't need to repeat the folder+file here (e.g /word-obfuscation/word-obfuscation.js), maybe put them inside a feature|replacement folder.

File: server/modules/blendsubs/blendsubs.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+    var words = lines[i].split(' ');
+
+    for (var j = 0; j < words.length; j++) {
+      if (words[j].length < 3 || /[A-Z<>]/.test(words[j])) {
+        continue;
+      }
+
+      if (Math.random() < threshold) {
```
Comment (danielfc) [inline server/modules/blendsubs/blendsubs.js]: here we call our replacement function (translate, obfuscate)...

File: server/modules/blendsubs/blendsubs.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+console.log('Reading str file: ' + file);
+fs.readFile(file, 'utf8', function(err, text) {
+  if (err) throw err;
+  console.log('Srt file read');
+
+  var lines = text.split('\n'),
+      changedWords = [];
+
+  for (var i = 0; i < lines.length; i++) {
+    if (lines[i].length < 10 || lines[i].indexOf('-->') > -1) {
```
Comment (danielfc) [inline server/modules/blendsubs/blendsubs.js]: line filtering

File: server/modules/blendsubs/blendsubs.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      continue;
+    }
+
+    var words = lines[i].split(' ');
+
+    for (var j = 0; j < words.length; j++) {
+      if (words[j].length < 3 || /[A-Z<>]/.test(words[j])) {
```
Comment (danielfc) [inline server/modules/blendsubs/blendsubs.js]: word filtering - if the word has at least one uppercase char or HTML tags, it won't be considered

**Comment by danielfc** [comment]
@joaomcar I managed to implement the idea for both replacement functions, please take a look.

**Comment by joaomcar** [comment]
Nice work.

**Pull request merged**